### PR TITLE
first basic mobile registration happy-path testcase

### PR DESCRIPTION
**Pull request by bekcunning**
basic registration testcase, enter all valid data and register success case.

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mobile/MobileSignupPageObject.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        waitForElementByElement(signupUsername);
+        signupUsername.sendKeys(username);
+    }
+
+    public void typePassword(String password) {
+        waitForElementByElement(signupPassword);
+        signupPassword.sendKeys(password);
+    }
+
+    public void typeBirthdate(String month, String day, String year) {
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mobile/MobileSignupPageObject.java]: You might get rid of this "void"s and go more chaining, 
```
public MobileSignupPageObject typePassword(String password) {
waitForElementByElement(signupPassword);
signupPassword.sendKeys(password);

return this;
}
```

File: src/test/java/com/wikia/webdriver/testcases/mobile/MobileSignupTests.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+/**
+ * Created by rcunningham on 6/30/15.
+ */
+public class MobileSignupTests extends NewTestTemplate{
+
+    Credentials credentials = config.getCredentials();
+
+    @Test(groups = {"MobileSignup_001", "MobileSignup", "Mobile"})
+    public void MobileSignup_001_signup() {
```
Comment (bkoval) [inline src/test/java/com/wikia/webdriver/testcases/mobile/MobileSignupTests.java]: I know this is a first one but d you guys call test cases in generic way or more descriptive, like MobileSignup_001_succesful_signup ? @bekcunning @ludwikkazmierczak

**Comment by bkoval** [comment]
Awesome work! Also, pinging @qaga to take a look :)

File: src/test/java/com/wikia/webdriver/testcases/mobile/MobileSignupTests.java (hunk 2 above)
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testcases/mobile/MobileSignupTests.java]: +1

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mobile/MobileSignupPageObject.java (hunk 1 above)
Comment (bekcunning) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mobile/MobileSignupPageObject.java]: I need to write other tests that will give invalid values for each of these items, which is why I created them each separately.  Do you have a better solution?

File: src/test/java/com/wikia/webdriver/testcases/mobile/MobileSignupTests.java (hunk 2 above)
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/testcases/mobile/MobileSignupTests.java]: Please don't use MobileSignup_001_ prefixes and "_" in method names. A method should be `succesfulSignup`

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mobile/MobileSignupPageObject.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+import java.nio.channels.ByteChannel;
+
+/**
+ * Created by rcunningham on 6/30/15.
+ */
+public class MobileSignupPageObject extends MobileBasePageObject {
+
+    public MobileSignupPageObject(WebDriver driver) { super(driver); }
+
+    // UI Mapping
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mobile/MobileSignupPageObject.java]: no need for comemnt

**Comment by ludwikkazmierczak** [comment]
LGTM

**Pull request merged**